docs/data-review-request.md

```diff
@@ -0,0 +1,105 @@
+
+<table>
+  <tr>
+    <td>Measurement Description</td>
+    <td>Data Collection Category</td>
+    <td>Tracking Bug #</td>
+  </tr>
+  <tr>
+    <td>reason - identifies the reason why this ping was sent, and can have the following values: progress (this is an intermediate measurement), final (this is the last measurement of the study) and error (this was triggered due to an error)</td>
+    <td>Category 1 “Technical data”</td>
+    <td>1432835</td>
+  </tr>
+  <tr>
+    <td>type -  the specific error type that triggered an error ping; this is optional, only available if reason = "error"; the field can have one of the following values: wrong-status (the remote server returned an error class other than 2xx), missing-performance (the performance data for the request is not available), request-aborted (the measurement was aborted to prevent the degradation of the connection) and request-error (there was a connection error)
+</td>
+    <td>Category 1 “Technical data”</td>
+    <td>1432835</td>
+  </tr>
+  <tr>
+    <td>latency - the round trip time to the CDN endpoint, in milliseconds, accurate to five thousandths of a millisecond</td>
+    <td>Category 1 “Technical data”</td>
+    <td>1432835</td>
+  </tr>
+  <tr>
+    <td>fileSize - the size, in bytes, of the remote sample file distributed by the CDN</td>
+    <td>Category 1 “Technical data”</td>
+    <td>1432835</td>
+  </tr>
+  <tr>
+    <td>connType - the detected connection type, either slow-2g, 2g, 3g, 4g or unknown</td>
+    <td>Category 1 “Technical data”</td>
+    <td>1432835</td>
+  </tr>
+  <tr>
+    <td>goodput - a vector with ordered tuples containing the time since the download started (in milliseconds, with the same precision as the latency) and the goodput in Mbps (or, more appropriately, Mibit/s);</td>
+    <td>Category 1 “Technical data”</td>
+    <td>1432835</td>
+  </tr>
+</table>
+
+6) How long will this data be collected?  Choose one of the following:
+
+This is scoped to a time-limited experiment/project until 4 weeks after the study is deployed.
+
+7) What populations will you measure?
+
+* Which release channels?
+
+Release
+
+* Which countries?
+
+USA, Germany, India
+
+* Which locales?
+
+All locales.
+
+* Any other filters?  Please describe in detail below.
+
+Random 0.5% sample.
+Windows only.
+Only Firefox builds with e10s enabled.
+
+8) If this data collection is default on, what is the opt-out mechanism for users?
+
+Users can unenroll through [about:studies](http://normandy.readthedocs.io/en/latest/user/actions/opt-out-study.html#about-studies). The study is automatically terminated and uninstalled if any of the following happens:
+
+- a slow connection is detected (goodput < 50Kbps);
+- 3 weeks have passed since the start of the study;
+- there was any error connecting to the CDN.
+
+9) Please provide a general description of how you will analyze this data.
+
+TODO
+
+10) Where do you intend to share the results of your analysis?
+
+TODO
```

Comment:
Currently, we aim to keep our results internal to Mozilla, so that colleagues can review methodology and discuss results. Presumably, we will discuss in internal meetings such as Data Club, Product Club etc.